admin_manual/configuration_server/occ_command.rst

```diff
@@ -1287,6 +1288,40 @@ authentication servers such as LDAP::
  | user directories | 2  |
  +------------------+----+
  
+Syncing User Accounts
+^^^^^^^^^^^^^^^^^^^^^
+
+This command syncs users stored in external backend services, such as _LDAP_, _Shibboleth_, and _Samba_, with ownCloud’s internal user database.
+
+.. note:: 
+   It’s also `one of the commands`_ that you should run on a regular basis to ensure that your ownCloud installation is running optimally.
+
+It’s not essential to run it regularly, unless you have a large number of users who change on a semi-regular basis.
+What’s more, the job is only necessary to pick up changes from a backend with users who don't log in regularly.
```

Comment:
I've just tested this and the answer is **no**.

Only the first login creates the user preemptively with populated properties in case it wasn't part of any "user:sync".
However subsequent logins do not sync anything. So if properties changed in LDAP like cn, display name (which I used to test), these will only be updated with the next `occ user:sync` call.

@settermjd please adjust